### PR TITLE
Fix invalid references in twisted

### PR DESCRIPTION
**Pull request by acabhishek942**
See : https://twistedmatrix.com/trac/ticket/8511

### CODE REVIEW

File: twisted/conch/ssh/userauth.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def ssh_USERAUTH_FAILURE(self, packet):
         removing methods that have already succeeded. We then call
         C{self.tryAuth} with the most preferred method.
 
-        @param packet: the L{MSG_USERAUTH_FAILURE} payload.
+        @param packet: the C{MSG_USERAUTH_FAILURE} payload.
```
Comment (acabhishek942) [inline twisted/conch/ssh/userauth.py:417]: 'MSG_USERAUTH_FAILURE' doesn't have dedicated documentation. So I changed markup from L{} to C{}

Comment (adiroiban) [inline twisted/conch/ssh/userauth.py:417]: Please try to add documentation :) ... it should be a good exercise of reading RFC documentation :)

**Comment by rodrigc** [comment]
I took a look at this link: https://pypi.python.org/pypi/pyobjc-framework-Cocoa
and one of the first things that it says there is that these packages do not provide documentation,
and people are supposed to refer to the Apple API documentation.  So the first approach you
did of converting L{CFRunLoop} to C{CFRunLoop} was probably OK.

Sorry to lead you in the wrong direction.

I am going to take the patch you submitted, fix it up a bit and get it in.

Thanks!

**Comment by adiroiban** [comment]
I think that using L{CFRunLoop} is ok and instead we should add better code when we generate the documentation so that we link to Apple API... so we can refer to the like L{apple.CFRunLoop}

**Comment by rodrigc** [comment]
The Apple API is not Python though, it is Objective-C, and users of the pyobjc library are told
to look at the Objective-C documentation.  For one file in the entire Twisted tree, I don't see it being worth jumping through all these hoops.  I don't think many people use the CFReactor, even on Mac.

**Pull request closed without merging**